authors/apps/authentication/views.py

```diff
@@ -153,3 +154,10 @@ def update(self, request, *args, **kwargs):
         serializer.save()
         msg = {'Message': 'Your password has been succesfully updated'}
         return Response(msg, status=status.HTTP_200_OK)
+
+
+class ListUserView(ListAPIView):
+    permission_classes = (IsAuthenticated,)
+    renderer_classes = (UserJSONRenderer,)
+    serializer_class = UserListSerializer
```

Comment:
Why did you, user list serializer not UserSerializer

Reply:
@akram256 , the UserSerializer serializer has a token field which is being used by other endpoints. over-riding it basing on which endpoint is accessing it would be hard. And yet we don't like to show the token when listing users.